src/components/SetupNFTClaimer.vue

```diff
@@ -0,0 +1,269 @@
+  if (contractInfo.value?.address) {
+    update(input.value);
+  } else {
+    deploy(input.value);
+  }
+}
+
+function toggleStatus() {
+  toggleMintStatus(!contractInfo.value.enabled);
+}
+
+function resetForm() {
+  input.value = pick(contractInfo.value, [
+    'maxSupply',
+    'formattedMintPrice',
+    'proposerFee',
+    'treasuryAddress'
+  ]);
+}
+
+function nextStep() {
+  if (!isViewOnly || !isValidJson) return forceShowError();
+  emit('next');
+}
+
+watch(
+  () => web3Account.value,
+  () => {
+    isSpaceController.value = !!web3Account.value;
+  },
+  { immediate: true }
+);
+
+watch(
+  () => init(props.space),
+  async () => {
+    input.value = contractInfo.value
+      ? pick(contractInfo.value, [
+          'maxSupply',
+          'formattedMintPrice',
+          'proposerFee',
+          'treasuryAddress'
+        ])
+      : {
+          maxSupply: '',
+          formattedMintPrice: '',
+          proposerFee: '',
+          treasuryAddress: ''
+        };
+
+    snapshotFee.value = await getSnapshotFee();
+  },
+  { immediate: true }
+);
+</script>
+
+<template>
+  <LoadingRow v-if="!inited" block />
+  <template v-else>
+    <BaseMessageBlock
+      v-if="isViewOnly"
+      class="md:mx-0"
+      level="info"
+      is-responsive
+    >
+      You are in view only mode, to modify space settings connect with a
+      controller wallet.
+    </BaseMessageBlock>
+    <BaseBlock v-if="contractInfo" class="mb-2 mt-3" title="Mint status">
+      <div class="flex gap-x-4">
+        <div class="grow">
+          <p>
+            At any time, you can disable/enable the minting status of this
+            space. Disabling will only prevent minting of future NFTs, and does
+            not affect existing tokens.
+          </p>
+        </div>
+        <div>
+          <BaseButton
+            class="whitespace-nowrap"
+            :variant="contractInfo.enabled ? 'danger' : undefined"
+            :disabled="isViewOnly"
+            @click="toggleStatus()"
+          >
+            {{ contractInfo.enabled ? 'Disable' : 'Enable' }} minting
+          </BaseButton>
+        </div>
+      </div>
+    </BaseBlock>
+    <form class="flex flex-col gap-y-3">
+      <BaseBlock title="SnapIt!">
+        <div class="flex w-full flex-col gap-y-3">
+          <BaseMessage level="info" class="mb-3">
+            Updates will only be applied to future proposals
+          </BaseMessage>
+          <TuneInput
+            v-model="input.maxSupply"
+            label="Max supply"
+            hint="Maximum number of NFTs per proposal"
+            placeholder="100"
+            min="1"
+            type="number"
+            :disabled="isViewOnly"
+            :error="validationErrors?.maxSupply"
+            autofocus
+            @update:model-value="value => (input.maxSupply = Number(value))"
+          />
+
+          <TuneInput
+            v-model="input.formattedMintPrice"
+            label="Mint price"
+            min="0"
```

Comment:
After removing the `min` property, I still have the issue with the backspace

Reply:
Ok weird, I can not reproduce it with BaseInput type number here https://tune-histoire.vercel.app/

Reply:
It's because of that I think
```
@update:model-value="
              value => (input.formattedMintPrice = Number(value))
            "
            ```

Reply:
Avoid double mutation by using `modelValue` instead of `v-model`

Reply:
By moving to `modal-value`, it indeed fix the bug, but also does not trigger validation anymore.

Every form where the `validationErrors` from validateForm helper is used, the TuneInput is using `v-model` instead of `model-value`

Reply:
This bug seems to be in other places, like when you add a strategy like `erc20-balance-of`, the decimals field also have this issue.